nx.json

```diff
@@ -3,6 +3,7 @@
   "affected": {
     "defaultBase": "master"
   },
+  "//": "",
```

Comment:
What's up with this?

Reply:
Wanted to invalidate the whole cache. It's not going to stay there